tests/test_minimal/test_tools/test_context_tools.py

```diff
@@ -197,3 +199,16 @@ def test_make_or_load_nwbfile_pass_nwbfile(self):
             nwbfile_out = io.read()
             assert "test1" in nwbfile_out.acquisition
             assert "test2" in nwbfile_out.acquisition
+
+
+def test_make_or_load_nwbfile_on_corrupt_file(tmpdir: Path) -> None:
```

Comment:
I think it would be beetter if this test were to import h5py explictly instead of doing it implicitly through

Reply:
Can you clarify?

Reply:
The h5py was not imported in the test module. But you added already. When I removed the h5py import in the PR  module the tests were failing.